### PR TITLE
Status indicator focus icon

### LINKED ISSUE #70: Focus Icon Not Intuitive, Pomo/Break Indicator Looks Like Buttons
Change the focus icon so users are more likely to understand its function by its shape. Current focus icon (magnifier) is not so intuitive.

Pomo/Break indicators look like buttons and are likely to cause confusions that they are clickable.

### PR DESCRIPTION
**Pull request by brooksniu**
==Changelog==

changed Pomo/Break indicator so they look less like clickables
changed focus icon so it is more intuitive

==Testing==

All tests passed

==Version Compatibility==

All version compatible

==Detail Description==

====Focus Icon on task list====
Before:
image
After:
image

====Pomo/Break Indicator====
Before: 
image
After:
image

===========Edit: Changed the Pomo/Break indicator such that both are displayed=========
image

### CODE REVIEW

**Comment by brooksniu** [comment]
There were too many merge conflicts when I tried to rebase and pull from origin, so I cloned the dev branch again, which resulted in all these commit histories. This branch is up-to-date with the dev branch. When merging we can "squash and merge" so these messy commit histories will be gone.

**Comment by brooksniu** [comment]
Assigned reviewers --- needs 2 approvals to merge.

**Review by kat066 — DISMISSED** [review]

File: source/src/index.html [hunk 1]
```diff
@@ -536,6 +539,7 @@ <h4 id="enable-disable-alarmsound" class="text" part="enable-disable-alarmsound"
             right:0;
             cursor: pointer;
             opacity: 0.33;
+            transition: transform .3s ease;
```
Comment (Juhmer-Tena) [inline source/src/index.html:568]: Is this already included in the html fragments?

**Review by Totay — DISMISSED** [review]
icons looks good!

File: source/src/index.html (hunk 1 above)
Comment (brooksniu) [reply source/src/index.html:568]: I only changed the css in index.html, just updated the html fragments!

**Review by Totay — APPROVED** [review]
lgtm

**Review by broiyn — APPROVED** [review]

**Pull request merged**